ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -242,4 +259,31 @@ private void createBanner() {
     public String toString() {
         return " [placementId=" + mPlacementId + " # uniqueRequestId=" + mUniquePubRequestId + " # hashcode=" + hashCode() + "] ";
     }
+
+    void attach() {
+        RelativeLayout layout = mAdLayout.get();
+        if (layout != null) {
+            if (mVungleBannerAd != null && mVungleBannerAd.getParent() == null) {
+                layout.addView(mVungleBannerAd);
+            }
+            if (mVungleNativeAd != null) {
+                View adView = mVungleNativeAd.renderNativeView();
+                if (adView != null && adView.getParent() == null) {
+                    layout.addView(adView);
+                }
+            }
+        }
```

Comment:
if layout null, should we destroy banner/MREC ad instance here?

Reply:
No, I think. I was trying to keep the old logic.
If we start finishing here it could also generate unwanted playAd callbacks